docs/high-level-overview.md

```diff
@@ -14,8 +14,15 @@ Conversely, there are many new ideas that exist in GEMD that are not captured by
 * Specified vs actual values
 * Ingredient information of input materials of a process
 
+![Object connectivity](img/GEMD-Object-Graph.png)
+
+Don't worry if that seems quite complicated right now.
+The details are broken out in the coming sections.
 
 ## How are Data Objects defined?
+A picture is worth a thousand words:
+
+![Material Graph](img/FORMULATIONS_ILLS_2.png)
```

Comment:
This image doesn't have any measurements in it.

Reply:
Supplied by Product Management.  I can look for better if this is a blocker, or we can move to better later if this is an improvement.

Reply:
I don't think we should block on it (consistent with @djack 's point about this being strictly better) but we do need a better image.